eo-maven-plugin/src/main/java/org/eolang/maven/SafeMojo.java

```diff
@@ -46,6 +48,7 @@
  * Abstract Mojo for all others.
  *
  * @since 0.1
+ * @checkstyle TooManyFields (300 lines)
  */
 abstract class SafeMojo extends AbstractMojo {
 
```

Comment:
@mximp as you can see here I suppressed this warning but it still occurs in log as an Error, I don't really get why